notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2600 @@
+{
```

Comment:
<p>What is your conclusion? That Flip Flakiness with optimal distance is the best, because it passes all test cases?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Yes, I added it.</p>